code/src/Components/Header.js

```diff
@@ -0,0 +1,14 @@
+import React from 'react'
+
+export const Header = () => {
+  return (
+    <div className="headerComponent">
+      <h1>SURVEY</h1>
+      <h2>Tell us how coding has been for you today</h2>
+      <span
+        className="computerIcon"
+        aria-label="Computer icon">&#128187;
+      </span>
```

Comment:
Using the screen reader, I noticed that it read out: "Computer icon, group, computer". So it actually seems that the SR (at least the one I'm using, actually is reading out the content of the Emoji 😮